daprdocs/content/en/getting-started/quickstarts/serviceinvocation-quickstart.md

```diff
@@ -178,7 +178,7 @@ npm install
 Run the `order-processor` service alongside a Dapr sidecar.
 
 ```bash
-dapr run --app-port 6001 --app-id order-processor --app-protocol http --dapr-http-port 3501 -- npm start
+dapr run --app-port 5001 --app-id order-processor --app-protocol http --dapr-http-port 3501 -- npm start
```

Comment:
This one is correct and should be 5001. Why these are all so different is strange!

Reply:
If the port is being passed as --app-port wouldn't it be more appropriate to read it from the APP_PORT environment variable in the code vs. fixing the documentatio?

Reply:
> If the port is being passed as --app-port wouldn't it be more appropriate to read it from the APP_PORT environment variable in the code vs. fixing the documentation?

Yes. Agreed on using APP_PORT. However the APP_PORT env variable is set by the Dapr CLI command and it would be best to make this consistent in the docs here.